src/components/ShareButton/styles.module.css

```diff
@@ -0,0 +1,47 @@
+@value layout: "../layout.css";
+
+@value sideSectionPadding from layout;
+@value darkSlateBlue from layout;
+@value teal: #1ABC9C;
+/* 
+ import "bootstrap";
+@import "_buttons.scss"; */
+
+.buttonSpacing {
+  padding: 1.7rem 0rem 0rem 16.03rem;
+}
+
+.buttonStyle {
```

Comment:
Can we make this name a bit more specific to what it is specifically in our case? Maybe something like `darkBlueButton` or `shareButton`?